### PR TITLE
fixed: holdbacks correctly display now

### PR DESCRIPTION
**Pull request by yspringfield**
Fixed: #7248

### CODE REVIEW

File: libs/contract/src/lib/contract/holdback/form/form.component.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ export class HolbackFormComponent {
   @Input() title: Movie;
   @Input() holdbacks: Holdback[] = [];
   @Output() holdbacksChange = new EventEmitter<Holdback[]>();
+  @ViewChild('formTable', { read: FormTableComponent }) formTable;
```
Comment (RemcoSimonides) [inline libs/contract/src/lib/contract/holdback/form/form.component.ts:21]: I think you can give it a type here `formTable: FormTableComponent`. Or is it already automatically typed?

File: libs/contract/src/lib/contract/holdback/form/form.component.ts [hunk 2]
```diff
@@ -43,11 +44,15 @@ export class HolbackFormComponent {
...
   save() {
     if (this.form.valid) {
+      this.ref.close(); //The position of the close before the emit is important.
       this.holdbacksChange.emit(this.form.value);
-      this.ref.close();
     }
   }
 
+  shouldDisable() {
+    return this.formTable.formItem && this.formTable.formItem?.dirty;
```
Comment (RemcoSimonides) [inline libs/contract/src/lib/contract/holdback/form/form.component.ts:53]: Only `return this.formTable.formItem?.dirty;` is ok

So now you can always add a holdback whenever it is dirty? Even if all data is empty?

File: libs/ui/src/lib/form/table/form-table.component.ts [hunk 3]
```diff
@@ -62,7 +62,7 @@ export class FormTableComponent<T> implements OnInit, OnDestroy {
   pageSize = 5;
   /* We need to keep track of the current page since it will affect the index that we are working on */
   pageConfig: PageState = { pageIndex: 0, pageSize: 5 };
-  formItem: FormEntity<EntityControl<T>, T>;
+  public formItem: FormEntity<EntityControl<T>, T>;
```
Comment (RemcoSimonides) [inline libs/ui/src/lib/form/table/form-table.component.ts:65]: It's public by default if you don't define 'private' or 'protected'. So no need to add this

File: libs/contract/src/lib/contract/holdback/form/form.component.ts (hunk 1 above)
Comment (yspringfield) [reply libs/contract/src/lib/contract/holdback/form/form.component.ts:21]: It's not typed. Just typed it.

**Review by RemcoSimonides — APPROVED** [review]

**Pull request merged**